meta-nemo/recipes-multimedia/pulseaudio/pulseaudio_%.bbappend

```diff
@@ -0,0 +1 @@
+RDEPENDS:pulseaudio-server:remove:pike= " pulseaudio-modules-droid "
```

Comment:
This should not be `pike`. Change it to `nemo`.

Reply:
omg... thanks for checking thoroughly!